hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieFileIndex.scala

```diff
@@ -267,10 +266,15 @@ case class HoodieFileIndex(spark: SparkSession,
         s"(isMetadataTableEnabled = $isMetadataTableEnabled, isColumnStatsIndexEnabled = $isColumnStatsIndexEnabled")
     }
   }
+
 }
 
 object HoodieFileIndex extends Logging {
 
+  def getBooleanConfigValue(sqlConf: SQLConf, options: Map[String, String], configKey: String, defaultValue: String) : Boolean = {
```

Comment:
Let's swap in SQLConf and opts order so that it matches the order of retrieving the values from them correspondingly